app/security/SecureActions.scala

```diff
@@ -112,7 +112,9 @@ trait SecureActions extends Silhouette[SecurityUser, SessionAuthenticator] {
 
   implicit def optionUserToUser(implicit u: CachedData): Option[CachedData] = Some(u)
 
-  implicit def userWithAppToOptionCachedUser(implicit u: CachedDataWithApp): Option[CachedData] = Some(CachedData(u.user, Some(u.application)))
+  implicit def userWithAppToOptionCachedUser(implicit u: CachedDataWithApp): Option[CachedData] = {
+    Some(CachedData(u.user, Some(u.application)))
```

Comment:
You are right: I added something and then removed it without taking off the curly brackets. I will do it.